### PR TITLE
ows-data replacement for ows-db dump in sandbox cluster

### PR DESCRIPTION
**Pull request by pindge**
- add more db roles
- add full suite of odc products for indexing from the same sqs 
- update image tags
- update role names

### CODE REVIEW

File: dags/sentinel_2_nrt/images.py [hunk 1]
```diff
@@ -5,8 +5,8 @@
...
 INDEXER_IMAGE = "opendatacube/datacube-index:0.0.9-9-g4aacd64"
 
-OWS_IMAGE = "opendatacube/ows:1.8.1"
-OWS_CONFIG_IMAGE = "pinjin/ows-config:s2_msiard"
+OWS_IMAGE = "opendatacube/ows:1.8.1-87-g9505e4d"
+OWS_CONFIG_IMAGE = "geoscienceaustralia/dea-datakube-config:1.5.2"
 
 
-EXPLORER_IMAGE = "opendatacube/dashboard:2.1.9"
+EXPLORER_IMAGE = "opendatacube/dashboard:2.2.1"
```
Comment (NikitaGandhi) [inline dags/sentinel_2_nrt/images.py:12]: EXPLORER_IMAGE = "opendatacube/explorer:2.2.1"

File: dags/sentinel_2_nrt/env_cfg.py [hunk 2]
```diff
@@ -9,8 +9,31 @@
 OWS_CFG_IMAGEPATH = "/opt/dea-config/prod/services/wms/ows/ows_cfg.py"
 
 # Process specific for sentinel indexing process
-INDEXING_PRODUCTS = "s2a_msiard_nrt_granule s2b_msiard_nrt_granule"
-ARCHIVE_PRODUCTS = "s2a_nrt_granule s2b_nrt_granule s2a_msiard_nrt_granule s2b_msiard_nrt_granule"
+ARCHIVE_PRODUCTS = "s2a_nrt_granule s2b_nrt_granule"
```
Comment (NikitaGandhi) [inline dags/sentinel_2_nrt/env_cfg.py:12]: can this be similar like INDEXING_PRODUCTS?

Comment (pindge) [reply dags/sentinel_2_nrt/env_cfg.py:12]: i actually need some python help, @whatnick, the actual list passed in is like this separated by a space,
but as the list goes infinitely long its impossible to read. so I used tuple, then when I actually use it in the code i then do 
`" ".join(list_name)`

is there a better way?

File: dags/sentinel_2_nrt/images.py (hunk 1 above)
Comment (pindge) [reply dags/sentinel_2_nrt/images.py]: thanks.

File: dags/sentinel_2_nrt/env_cfg.py (hunk 2 above)
Comment (alexgleith) [reply dags/sentinel_2_nrt/env_cfg.py:12]: That's a good way.

File: dags/sentinel_2_nrt/sentinel_2_nrt_indexing.py [hunk 3]
```diff
@@ -48,22 +50,23 @@
...
 
+record_path_list_with_prefix = ['--record-path ' + path for path in PRODUCT_RECORD_PATHS]
 
 INDEXING_BASH_COMMAND = [
     "bash",
     "-c",
     dedent(
         """
-            sqs-to-dc %s "%s" --skip-lineage --allow-unsafe --record-path "L2/sentinel-2-nrt/S2MSIARD/*/*/ARD-METADATA.yaml";
+            sqs-to-dc %s "%s" --skip-lineage --allow-unsafe %s;
```
Comment (alexgleith) [inline dags/sentinel_2_nrt/sentinel_2_nrt_indexing.py:66]: We use f strings, usually. Much cleaner than `%` for string building.

Comment (pindge) [reply dags/sentinel_2_nrt/sentinel_2_nrt_indexing.py]: cool, was so stuck with the idea of using `dedent` forgot to get creative about `fstring`, thanks

**Pull request merged**